src/main/groovy/edu/oregonstate/mist/api/Resource.groovy

```diff
@@ -81,7 +81,7 @@ abstract class Resource {
      * @param message
      * @return bad request response builder
      */
-    protected static ResponseBuilder badRequest(String message) {
+    protected static ResponseBuilder badRequest(String message = '') {
```

Comment:
Why an empty string as a default? I think we should require some sort of message here. An empty string isn't too helpful of an error message.

Reply:
The elvis operator handles this string in Error.groovy:

https://github.com/osu-mist/web-api-skeleton/blob/8bdfdf707ec2fa7091c70425c89c45f73965da2a/src/main/groovy/edu/oregonstate/mist/api/Error.groovy#L44

The default message in resource.properties is returned in the response.

A null string is handled the same though, and it is confusing to have a default string value in this class. I'll get those removed.